bindings/python/synthizer.pyi

```diff
@@ -0,0 +1,260 @@
+from enum import Enum
+from typing import Iterator, List, Optional, Tuple, Union
+
+StringOrBytes = Union[str, bytes]
+Double3 = Tuple[float, float, float]
+Double6 = Tuple[float, float, float, float, float, float]
+EventSourceType = Union['Source', 'Generator']
+
+
+class LoggingBackend(Enum):
+    STDERR: int
+
+
+STDERR: LoggingBackend = LoggingBackend.STDERR
+
+
+class LogLevel(Enum):
+    DEBUG: int
+    ERROR: int
+    INFO: int
+    WARN: int
+
+
+DEBUG: LogLevel = LogLevel.DEBUG
+INFO: LogLevel = LogLevel.INFO
+ERROR: LogLevel = LogLevel.ERROR
+WARN: LogLevel = LogLevel.WARN
+
+
+class DistanceModel(Enum):
+    NONE: int
+    LINEAR: int
+    EXPONENTIAL: int
+    INVERSE: int
+
+
+NONE: DistanceModel = DistanceModel.NONE
+LINEAR: DistanceModel = DistanceModel.LINEAR
+EXPONENTIAL: DistanceModel
+INVERSE: DistanceModel
+
+
+class NoiseType(Enum):
+    FILTERED_BROWN: int
+    UNIFORM: int
+    VM: int
+
+
+FILTERED_BROWN: NoiseType = NoiseType.FILTERED_BROWN
+UNIFORM: NoiseType = NoiseType.UNIFORM
+VM: NoiseType = NoiseType.VM
+
+
+class PannerStrategy(Enum):
+    HRTF: int
+    STEREO: int
+
+
+HRTF: PannerStrategy = PannerStrategy.HRTF
+STEREO: PannerStrategy = PannerStrategy.STEREO
+
+# Initialization functions:
+
+
+def initialized() -> Iterator[None]: ...
```

Comment:
This is a context manager, not an iterator.

Reply:
I was working according to [https://stackoverflow.com/questions/49733699/python-type-hints-and-context-managers](this question), where it mentions typing the function as an iterator.

That said, there is also `typing.ContextManager`, which may be more appropriate. I'll make that change.